packages/tooltip/test/tooltip.test.js

```diff
@@ -320,4 +320,57 @@ describe('vaadin-tooltip', () => {
       expect(overlay.opened).to.be.true;
     });
   });
+
+  describe('manual', () => {
+    let target;
+
+    beforeEach(() => {
+      target = document.createElement('input');
+      document.body.appendChild(target);
```

Comment:
Why not use `target = fixtureSync('<input>');` here (and drop the `afterEach`)?

Reply:
Good point, thanks. Done.